docs/physics/state.rst

```diff
@@ -0,0 +1,45 @@
+.. _state:
```

Comment:
This tag will conflict with `docs/util/state.rst`, please change the label.

Reply:
What was done to resolve this?

Reply:
removed